File: backend/DataAccess/DaoManual/EUC機能/tm_euyosikisyosaiDto.cs
// *******************************************************************
// 業務名称　: 健康管理システム
// 機能概要　: DTO定義
//             EUC様式詳細マスタ
// 作成日　　: 2023.01.23
// 作成者　　: 屠
// 変更履歴　:
// *******************************************************************
namespace BCC.Affect.DataAccess
{
    public class tm_euyosikisyosaiDto : DaEntityModelBase
    {
		public const string TABLE_NAME = "tm_euyosikisyosai";
		public string rptid { get; set; }                         //帳票ID
		public string yosikiid { get; set; }                      //様式ID
		public int yosikieda { get; set; }                        //様式枝番
		public string yosikinm { get; set; }                      //様式名
		public Enum様式種別 yosikisyubetu { get; set; }                 //様式種別
		public Enum様式種別詳細 yosikikbn { get; set; }                     //様式種別詳細
		public bool meisaiflg { get; set; }                       //明細有無
		public string? meisaikoteikbn { get; set; }               //行（列）固定区分
		public Enum様式作成方法 yosikihouhou { get; set; }                  //様式作成方法
		public string? datasourceid { get; set; }                 //データソースID
		public int? himozukejyokenid { get; set; }                //様式紐付け条件ID
		public string? sql { get; set; }                          //SQL
		public string? procnm { get; set; }                       //プロシージャ名
		public bool distinctflg { get; set; }                     //重複データ排除フラグ
		public bool nulltozeroflg { get; set; }                   //空白値ゼロ出力フラグ
		public int? startrow { get; set; }                        //テンプレート明細開始行
		public int? loopmaxrow { get; set; }                      //１ページあたりの最大行数
		public int? looprow { get; set; }                         //1明細あたりの行数
		public int? startcol { get; set; }                        //明細開始列数
		public int? loopmaxcol { get; set; }                      //１ページあたりの最大列数
		public int? loopcol { get; set; }                         //1明細あたりの列数
		public string reguserid { get; set; }                     //登録ユーザーID
		public DateTime regdttm { get; set; }                     //登録日時
		public string upduserid { get; set; }                     //更新ユーザーID
		public DateTime upddttm { get; set; }                     //更新日時
    }
}

File: GJSBackend/BusinessService/Db/DaoManual/EUC機能/tm_euyosikisyosaiDto.vb
' *******************************************************************
' 業務名称　: 互助防疫システム
' 機能概要　: DTO定義
'             EUC様式詳細マスタ
' 作成日　　: 2024.07.23
' 作成者　　: 
' 変更履歴　:
' *******************************************************************
Namespace Jbd.Gjs.Db
    Public Class tm_euyosikisyosaiDto
        Inherits DaEntityModelBase
        Public Const TABLE_NAME As String = "tm_euyosikisyosai"
        Public Property rptid As String                         '帳票ID
        Public Property yosikiid As String                      '様式ID
        Public Property yosikieda As Integer                        '様式枝番
        Public Property yosikinm As String                      '様式名
        Public Property yosikisyubetu As Enum様式種別                 '様式種別
        Public Property yosikikbn As Enum様式種別詳細                     '様式種別詳細
        Public Property meisaiflg As Boolean                       '明細有無
        Public Property meisaikoteikbn As String               '行（列）固定区分
        Public Property yosikihouhou As Enum様式作成方法                  '様式作成方法
        Public Property datasourceid As String                 'データソースID
        Public Property himozukejyokenid As Integer                '様式紐付け条件ID
        Public Property sql As String                          'SQL
        Public Property procnm As String                       'プロシージャ名
        Public Property distinctflg As Boolean                     '重複データ排除フラグ
        Public Property nulltozeroflg As Boolean                   '空白値ゼロ出力フラグ
        Public Property startrow As Integer                        'テンプレート明細開始行
        Public Property loopmaxrow As Integer                      '１ページあたりの最大行数
        Public Property looprow As Integer                         '1明細あたりの行数
        Public Property startcol As Integer                        '明細開始列数
        Public Property loopmaxcol As Integer                      '１ページあたりの最大列数
        Public Property loopcol As Integer                         '1明細あたりの列数
        Public Property reguserid As String                     '登録ユーザーID
        Public Property regdttm As Date                     '登録日時
        Public Property upduserid As String                     '更新ユーザーID
        Public Property upddttm As Date                     '更新日時
    End Class
End Namespace
